docs/contribute/01_environment.qmd

```diff
@@ -58,7 +62,10 @@ for manager, params in managers.items():
     print(f"1. Install `gh`")
     print()
     print("    ```sh")
-    print(f"    {manager} install -c conda-forge gh")
+    if manager == "conda" or manager == "mamba":
+        print(f"    {manager} install -c conda-forge gh")
+    elif manager == "pixi":
+        print(f"    {manager} global install gh")
```

Comment:
I didn't do this because I don't use `gh`, but wanted to keep things consistent...